### PR TITLE
Added payment model

### PR DESCRIPTION
**Pull request by holsoni**

### CODE REVIEW

**Review by CodeWebWeaver — DISMISSED** [review]

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+databaseChangeLog:
+  - include:
+        file: db/changelog/changes/02.10.create-rentals-table.yaml
+  - include:
+      file: db/changelog/changes/02.10.create-payment-table.yaml
```
Comment (Dimagaa) [inline src/main/resources/db/changelog/db.changelog-master.yaml]:
```suggestion
      file: db/changelog/changes/02.10.2023.create-rentals-table.yaml
  - include:
      file: db/changelog/changes/02.10.2023.create-payment-table.yaml
```

**Review by Dimagaa — DISMISSED** [review]
Well Done, Just minor fixes

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+databaseChangeLog:
+  - include:
+        file: db/changelog/changes/02.10.23.create-rentals-table.yaml
```
Comment (Dimagaa) [inline src/main/resources/db/changelog/db.changelog-master.yaml]:
```suggestion
        file: db/changelog/changes/02.10.2023.create-rentals-table.yaml
```

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+  - include:
+      file: db/changelog/changes/02.10.23.create-payment-table.yaml
```
Comment (Dimagaa) [inline src/main/resources/db/changelog/db.changelog-master.yaml]:
```suggestion
      file: db/changelog/changes/02.10.2023.create-payment-table.yaml
```

**Review by Dimagaa — DISMISSED** [review]
Approved!

**Review by Dimagaa — APPROVED** [review]

**Review by CodeWebWeaver — APPROVED** [review]

**Pull request merged**